app/views/firms/partials/_header.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="firm__header">
   <%= heading_tag(@firm.name, level: 2, class: 'firm__name') %>
   <% if search_form.face_to_face? %>
-    <%= render partial: 'search/partials/firm/adviser_distance', object: @firm, as: :firm, locals: { search_form: search_form }%>
+    <%= render partial: 'search/partials/firm/adviser_distance', locals: { distance: @firm.closest_adviser }%>
```

Comment:
I was a little confused by this change when I first looked at it. The simplification here and the corresponding change in `helper.closest_adviser_text` is to ease the reuse of `_adviser_distance.html.erb` within `_recently_visited.html.erb`.